### PR TITLE
Adds a Discord button & stylizes the Karma Shop and Donate buttons

### PR DESCRIPTION
**Pull request by Ty-Omaha**
jwfqtuj

Adds the discord button in discord colors
Makes the karma shop in bold
Makes the donate button green, and bold

It's easier to read with them bolded, I think at least.

### CODE REVIEW

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -85,6 +85,7 @@
 	var/githuburl = "http://example.org"
 	var/donationsurl = "http://example.org"
 	var/repositoryurl = "http://example.org"
+	var/discordurl ="http://example.org"
```
Comment (Birdtalon) [inline code/controllers/configuration.dm]: Spacing 🔨

**Comment by IrkallaEpsilon** [comment]
Looks good

**Comment by Ty-Omaha** [comment]
@Crazylemon64 done

**Comment by variableundefined** [comment]
Tested, LGTM. Just need someone to do the config changes.

**Comment by Crazylemon64** [comment]
Config done. Here we go!

**Pull request merged**